src/main/java/carpet/script/value/EntityValue.java

```diff
@@ -1668,12 +1690,57 @@ else if (v instanceof ListValue)
             }
         });
 
+        put("blue_skull",(e,v)->{
+            if (e instanceof WitherSkull w){
+                w.setDangerous(v.getBoolean());
+            }
+            
+        });
+        put("offering_flower",(e,v)->{
+            if (e instanceof IronGolem ig){
+                ig.offerFlower(v.getBoolean());
+            }
+            
+        });
+        put("item", (e, v) -> {
+                ItemStack item=getItemStackFromValue(v).copy();
+                if(e instanceof ItemEntity iteme)            
+                    iteme.setItem(item);
+                if(e instanceof ItemFrame iteme)
+                    iteme.setItem(item);  
+        });
         // "dimension"      []
-        // "item"           []
         // "count",         []
         // "effect_"name    []
     }};
 
+    static public ItemStack getItemStackFromValue(Value value){
```

Comment:
Question (I don't know), isn't there something to make this already from a triple? I'd assume there'd be other functions doing similar stuff.

Reply:
Note that I haven't really reviewed this method yet.

Reply:
imo this method fits in better in ValueConversions

Reply:
it does fit better in value conversions - not in entity code

Reply:
value conversions in general convert the other way. There is NBTSerializableValue.parseItem that returns one without count, which could be used instead of this method (it does the same thing, just cached)

Reply:
this method IS using said NBTSerializableValue.parseItem to do so.
and i move it to ValueConversions as every one thinks it fit there better.